File: Programas em Phyton/Ex18.py
p = float(input('Peso: '))
a = float(input('Altura: '))
sexo = input('Sexo [F] ou [M]: ')
R = p/(a*a)
if(sexo=="M"):
    if(R<20):
        print("Abaixo do peso")
    elif(20<=R<25):
        print("Peso ideal")
    else:
        print("Acima do peso")
else:
    if(R<19):
        print("Abaixo do peso")
    elif(19<=R<24):
        print("Peso ideial")
    else:
        print("Acima do peso")

File: Programas em C/Ex18.c
#include<stdio.h>
int main(int argc, char const *argv[])
{
    float P, A, X;
    char S, M, F;
    printf("Peso: ");
    scanf("%f", &P);

    printf("Sexo: ");
    scanf("%s", &S);

    printf("Altura: ");
    scanf("%f", &A);

    X = P / (A*A);
    S == M || S == F;

    if(S == F){
        if(X<19){
            printf("Abaixo do peso");
        }
        else if(X<24){
            printf("Peso ideal");
        }
        else{
            printf("Acima do peso");
        }
    } 
    else{
        if(X<20){
            printf("Abaixo do peso");
        }
        else if(X<25){
            printf("Peso ideal");
        }
        else{
            printf("Acima do peso");
        }
    }
    return 0;
}
